nix/modules/scripts.nix

```diff
@@ -60,9 +61,13 @@
           git checkout flake.lock
         else
           git add flake.lock
+          ANY_CHANGED=1
         fi
 
-        git commit -m "chore(flakes): update $VERSIONS_DIR"
+        if [[ $ANY_CHANGED -eq 1 ]]; then
```

Comment:
reads like a proxy variable to the status of `git diff --staged --exit-code` to me, WDYT?

Reply:
I think it's simpler not to touch that, I've done it but dealing with exit codes is harder to read IMO